### PR TITLE
#146 allow using TOKENINFO_URL env var (OAuth2 token info)

### LINKED ISSUE #146: Use "TOKENINFO_URL" environment variable as fallback
We decided to standardize the OAuth 2 token info URL configuration: we are now using the `TOKENINFO_URL` environment variable which should be supported by Connexion.

See also https://github.com/zalando-stups/taupage/issues/177

### PR DESCRIPTION
**Pull request by hjacobs**
Fixes #146

### CODE REVIEW

File: connexion/decorators/security.py [hunk 1]
```diff
@@ -28,6 +29,20 @@
...
+def get_tokeninfo_url(security_definition):
+    '''
+    :type security_definition: dict
+    :rtype: str
+
+    >>> get_tokeninfo_url({'x-tokenInfoUrl': 'foo'})
+    'foo'
+    '''
+    token_info_url = security_definition.get('x-tokenInfoUrl') or \
+        os.environ.get('HTTP_TOKENINFO_URL') or \
```
Comment (rafaelcaricio) [inline connexion/decorators/security.py]: Maybe we should log a warning message notifying the user -- if the `HTTP_TOKENINFO_URL` is set -- that they should use `TOKENINFO_URL`?

Comment (hjacobs) [inline connexion/decorators/security.py]: Probably a good idea, so we can deprecate it in some later version.

**Comment by hjacobs** [comment]
@rafaelcaricio updated with warning log :smile:

**Comment by rafaelcaricio** [comment]
Great! LGTM! :smile:

**Pull request merged**